graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasJanusElement.java

```diff
@@ -200,8 +204,8 @@ public String getIdForDisplay() {
 
         List<String> value = getListProperty(propertyName);
 
-        if (value.isEmpty()) {
-            return (List<V>)value;
+        if (value == null || value.isEmpty()) {
+            return value == null ? null : new ArrayList<>();
```

Comment:
null check again here is not needed.

if (CollectionUtils.isEmpty(value)) {
  return value;
}